doc/api/next_api_changes/2019-04-11-AL.rst

```diff
@@ -0,0 +1,5 @@
+Stricter parsing of the Composites section of AFM files
```

Comment:
The PR title and commit message suggest that it does not have any effect on running code. However, here you state an API change. I'd feel much safer if this was a separate PR, not least because this PR is large.